indico/modules/events/abstracts/templates/reviewing/judgment.html

```diff
@@ -163,11 +163,11 @@
                                                 {%- endtrans %}
                                             </div>
                                             <a href="#abstract-review-{{ review.id }}"
-                                               class="track-review-details-link js-highlight-review">
+                                               class="group-review-summary__link js-highlight-review">
```

Comment:
This is his way to slooooooowly move to BEM :snail:
Add one class here, one class there, no-one will notice! haha!